src/nft/components/common/SortDropdown/SortDropdown.tsx

```diff
@@ -70,51 +72,56 @@ export const SortDropdown = ({
         width={inFilters ? 'full' : 'inherit'}
         onClick={toggleOpen}
         cursor="pointer"
-        className={clsx(isOpen && !mini && styles.activeDropdown)}
+        className={isCollectionNftsLoading ? styles.isLoadingDropdown : clsx(isOpen && !mini && styles.activeDropdown)}
```

Comment:
The filter and search loading components overlap
![Screen Shot 2022-09-28 at 11 59 40 ](https://user-images.githubusercontent.com/11512321/192871078-5deee941-b107-4ff0-80de-55c1986967a6.png)